### PR TITLE
Smooth scrolling, navigation anchors and page titles

### PR DESCRIPTION
**Pull request by Dygerydoo**
I added a smooth scrolling effect when you click on the navigation menu links, also I added the function to display the page title depending on which page you are.

### CODE REVIEW

File: app/assets/javascripts/psique/documentation.js.coffee [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ------------------------------
+# Smooth Scrolling for Styleguide
+# ------------------------------
+$(document).ready ->
+  $('a').click ->
```
Comment (Fernan2) [inline app/assets/javascripts/psique/documentation.js.coffee]: All the links will be animated?  :-/

I usually prefer to keep the standard behavior when no class is specified and use a class to add customized behaviors...

Comment (Dygerydoo) [inline app/assets/javascripts/psique/documentation.js.coffee]: @Fernan2 Solved

**Comment by Fernan2** [comment]
+1

**Comment by d2bit** [comment]
:+1:

**Pull request merged**